recipes/heliopy/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "heliopy" %}
+{% set version = "0.1.3" %}
+{% set sha256 = "d1c0bf43f156ef795cab581fd8a57c2d789ba191c2909cc4b94ea3a2777efc18" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [py<35]
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
```

Comment:
It looks like some additional packages are required as `run` requirements:

https://github.com/heliopython/heliopy/blob/master/setup.py#L18-L21